### DISCUSSION TITLE
Relase notes after 2.5.0

### DISCUSSION THREAD

**Discussion by sanfair** [ueberdosis/tiptap#5370] [General Questions] [answered]
Hi, I have a question about the updated release process.
How and where can I conveniently review the changes in the packages I rely on (@tiptap/*)?
Please compare my current flow and explain how I should adjust it.

I use Dependabot to manage dependencies on my project. It conveniently generated a PR description with:
1. Name of the package
2. Release notes (if a release exists in GitHub)
3. Changelog (if CHANGELOG.md exists in the package's repo)
4. Commits - in case I want to review changes

I added all TipTap packages in one group because you have a mono repo and one release for all packages, so basically, it was repeating all that information for each package.
<details>
  <summary>PR's description example</summary>

  ![image](https://github.com/user-attachments/assets/1d536e46-c7de-4333-aea4-5629c7a9589b)
</details>

<details>
  <summary>PR's release notes example</summary>

  ![image](https://github.com/user-attachments/assets/382a5f3c-4614-4dd9-9b2d-eca23bc8dd48)
</details>

So, my current process is: 
1. Review `Release notes` or `Changelog`
2. Optionally, check the issue/commit with changes if it draws my attention
3. Review changes in `package.json` and `package-lock.json`
4. Approve and merge

---

<details>
  <summary><b>But since 2.5.0, it is just nonsense. (Relase notes after 2.5.0)</b></summary>

  ![image](https://github.com/user-attachments/assets/93f9716e-eb00-424d-8907-a7df87ec5e47)
</details>

<details>
  <summary>It is the same way that releases are shown on GitHub.</summary>

 
![image](https://github.com/user-attachments/assets/5da93cb7-946d-45cc-807d-de793dc09320)

</details>

<details>
  <summary>I can't tell if the release notes are relevant for <code>@tiptap/extension-text.</code></summary>

  ![image](https://github.com/user-attachments/assets/10ad4772-cfb9-4ad0-b566-cdeab7f24032)
</details>

<details>
  <summary>Changelogs are useless too.</summary>

  ![image](https://github.com/user-attachments/assets/184c0bd7-c8f8-4bc3-944b-c008f585b2b8)
</details>

---

It needs only one commit to have a payload that triggers something unexpected. It is not getting easier to catch this during code review.

<details>
  <summary>Also, it would be great if you could address "Maintainer changes"</summary>

  ![image](https://github.com/user-attachments/assets/3e84b230-af4e-4c19-b365-d25351c09ea2)
</details>

**Comment by nperez0111** [comment, marked as answer]
I appreciate your feedback here (well formatted & detailed too!)

We have been receiving a bunch of feedback around our recent usage of changesets for releasing things and we are working towards a possible solution. There is a tension between how we previously have packaged things (all being one version) and how we may want to package things in the future (independent versioning). For now we don't have a great solution for how we will achieve this but are in talks about it internally.

As for "Maintainer changes", Hi I'm Nick and a relatively new employee at Tiptap, it was published under my account since I was working on the pipelines for releasing. Not sure what there is to address beyond that.
